admin/class-smaily-for-wp-admin.php

```diff
@@ -67,6 +67,57 @@ public function enqueue_scripts() {
 		}
 	}
 
+	/**
+	 * Verify plugin was updated via transient and trigger DB upgrades.
+	 *
+	 * @since 3.0.0
+	 */
+	public function listen_for_upgrade_transient() {
+		if ( get_transient( 'smailyforwp_plugin_updated' ) !== true ) {
+			return;
+		}
+
+		$plugin_version = SMLY4WP_PLUGIN_VERSION;
+		if ( $plugin_version === get_option( 'smailyforwp_db_version' ) ) {
+			return delete_transient( 'smailyforwp_plugin_updated' );
+		}
+
+		if ( version_compare( $plugin_version, '3.0.0', '=' ) ) {
+			require_once SMLY4WP_PLUGIN_PATH . '/migrations/upgrade-3-0-0.php';
```

Comment:
Please change migration file name to `upgrade-3_0_0.php`. This way we can easily extract version info from filename in the future, if necessary.

Reply:
7f64216